mods/stairs/README.txt

```diff
@@ -7,3 +7,18 @@ Authors of source code
 Originally by Kahrl <kahrl@gmx.net> (LGPLv2.1+) and
 celeron55, Perttu Ahola <celeron55@gmail.com> (LGPLv2.1+)
 Various Minetest developers and contributors (LGPLv2.1+)
+
+Authors of media (textures)
+---------------------------
+
+Textures
+--------
+
+Copyright (c) 2017 Shara RedCat (CC BY-SA 3.0)
+ stairs_obsidian_glass_outer_stairside.png
+ stairs_obsidian_glass_stairside.png
+
```

Comment:
2018, and they are just as derived as your own textures.

Reply:
I will fix